demo/src/index.js

```diff
@@ -13,12 +13,13 @@ const Demo = () => (
       className="boss"
       style={{
         color: '#333',
-        textDecoration: 'none'
+        textDecoration: 'none',
+        top: '200px'
```

Comment:
Seems like a lot. Can you explain why it's so much?

Reply:
you mean the `top` value ?, just to make sure the complete `react better mail to` is visible

Reply:
You mean the `top` value it's just to make sure the react `better mail to` component is visible.